evm-ds/src/convert.rs

```diff
@@ -178,3 +181,153 @@ impl From<(Bytes, Bytes)> for EvmProto::Storage {
         result
     }
 }
+
+fn as_string(opcode: &Opcode) -> String {
```

Comment:
This should rather be done as `impl fmt::Display for Opcode`

Reply:
which you can as well implement by calling into fmt::Debug imlementation that Rust already provides for enums

Reply:
Indeed, as discussed in chat, outside evm crate you can't do it, please ignore the above.